### PR TITLE
Add Scientific Writing

### PR DESCRIPTION
**Pull request by maehr**
**[Insert URL to the list here]**

https://github.com/writing-resources/awesome-scientific-writing

**[Explain what this list is about and why it should be included here]**

Open source tools for painless academic writing.

This should be included, because academic writing is a hassle.

PRs:

- https://github.com/sindresorhus/awesome/pull/1735
- https://github.com/sindresorhus/awesome/pull/1635
- https://github.com/sindresorhus/awesome/pull/1644

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

File: readme.md [hunk 1]
```diff
@@ -570,6 +570,7 @@
 
 ## Higher Education
 
+- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Open source tools for painless academic writing.
```
Comment (ibnesayeed) [inline readme.md]: The word "painless" in the description is subjective. Also, on the list page itself or its contribution guidelines I did not see anything that enforces only open source tools to be added to it.

File: readme.md [hunk 2]
```diff
@@ -570,6 +570,7 @@
 
 ## Higher Education
 
+- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Tools for distraction free academic writing with Markdown.
```
Comment (ibnesayeed) [inline readme.md]: PR Template discourages use of terms like "resources" and "tools" in the description as awesome lists are primarily tools and resources, so it is obvious. I would perhaps change it to something like:

```suggestion
- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Distraction-free academic writing with Markdown.
```

**Comment by ibnesayeed** [comment]
Another PR guideline suggests that:

> Your entry should be added at the bottom of the appropriate category.

However, you seem to have added it at the top to maintain sort order.

**Comment by thecoder8890** [comment]
Good to go

**Comment by tobiasvl** [comment]
Looks good! But I think there's one last, horned guideline that hasn't been followed here yet?

There are many entries here that require specific knowledge to understand, but it's hard for me to tell whether anyone who is in academia would know the context (does everyone know what CSL stands for?). However, the following description seems vague and has a spelling error:

> CSL Language and Editor - If you don not like my style, create your own.

File: readme.md (hunk 1 above)
Comment (maehr) [reply readme.md]: Please rereview my PR

**Comment by ashwinvis** [comment]
FYI, there is a similar list that I have been grooming for submission here: https://github.com/ashwinvis/awesome-scientific-writing

**Comment by maehr** [comment]
> FYI, there is a similar list that I have been grooming for submission here: https://github.com/ashwinvis/awesome-scientific-writing

I like your list. We could merge our lists or further differentiate. My focus is clearly Pandora Markdown.

**Comment by ashwinvis** [comment]
@maehr I agree with that proposal, and am willing to merge. You have some things that I left out. Shall I move mine to a common organization, so we can work together and make PRs to merge?

**Comment by maehr** [comment]
> @maehr I agree with that proposal, and am willing to merge. You have some things that I left out. Shall I move mine to a common organization, so we can work together and make PRs to merge?

Yes, perfectly. I am looking forward to streamline the list together.

**Comment by maehr** [comment]
@ashwinvis and I merged our lists. We are ready to go.

**Comment by maehr** [comment]
unicorn

**Comment by ashwinvis** [comment]
@ibnesayeed @thecoder8890 Do you approve the latest changes?

**Comment by ibnesayeed** [comment]
LGTM!

**Comment by ashwinvis** [comment]
@sindresorhus, we have gone through the checklist in detail and got thumbs up from both reviewers.

**Comment by sindresorhus** [comment]
> Consistent and correct naming. For example, Node.js, not NodeJS or node.js.

I see `Markdown` being written as `markdown` in at least one of the descriptions.

I also see some typos in the descriptions. Please go through all the descriptions and try to improve them.

---

> Do not add the license name or text to the readme. GitHub already shows the license name at the top of the repo.

**Comment by sindresorhus** [comment]
I decided to rename `master` branch to `main`, and apparently that permanently closes all PRs. Ugh. Sorry about that. Can you reopen this PR targeting the `main` branch? Include a link to this PR in the new one, so we can quickly see what was discussed previously.

**Pull request closed without merging**